packages/Sortable/src/SortableContext.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+const SortableContext = React.createContext();
```

Comment:
🤔 no sure if this worth to have their own file for 1 line of code. My `giveAFu`  thermometer is low on this.

Reply:
This pattern is needed because otherwise there will be a dependency cycle that violates the `import/no-cycle` eslint rule.

Sortable exports context, and imports Item.  Item imports context and exports itself for Sortable. 
Sortable depends on Item and Item depends on Sortable.  And on and on it goes, like:

![](https://media1.giphy.com/media/2JeYuSGvqapfa/giphy.gif?cid=790b76115d0c733e4877323477a60c67&rid=giphy.gif)

Reply:
ohhhh now I understand why I hate that rule `import/no-cycle`